docs/src/main/asciidoc/index-docinfo.xml

```diff
@@ -1,8 +1,8 @@
 <productname>Spring Cloud GCP</productname>
 <releaseinfo>{project-version}</releaseinfo>
 <copyright>
-	<year>2017-2019</year>
-	<holder>Pivotal Software, Inc.</holder>
+	<year>2017-2021</year>
+	<holder>Google LLC</holder>
```

Comment:
```suggestion
	<holder>Google, LLC</holder>
```

Reply:
I thought so too, but I see it used without the comma everywhere.